doc/sphinx/source/extensions.rst

```diff
@@ -11,6 +16,14 @@ This extension provides a way to to indicate
 a buffer which will hold the meaningful
 bytes of another buffer, after kernel execution.
 
-Full specification can be found on:
 
-https://www.khronos.org/registry/OpenCL/
+cl_khr_command_buffer
+~~~~~~~~~~~~~~~~~~~~~~~
+
+This extension provides a way to record a sequence
+of OpenCL commands that can be executed as a single
+invocation. Command parameters are validated and
```

Comment:
In the current PoCL implementation, ...     So mention what is currently "cached" (basically the error checking).